mappings/net/minecraft/client/MinecraftClient.mapping

```diff
@@ -470,6 +471,8 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	METHOD method_41733 forceStopRecorder ()V
 	METHOD method_41734 getGpuUtilizationPercentage ()D
 	METHOD method_41735 createIntegratedServerLoader ()Lnet/minecraft/class_7196;
+	METHOD method_43587 setCrashReportSupplierWithoutDetails (Lnet/minecraft/class_128;)V
```

Comment:
Not sure I understand what you're saying here...

Reply:
This one is called with reports that already have their details filled. The other one is called with a bare report so that report has to have its detail filled.

Reply:
I think Shnupbups was not sure what "swap their names" means. The name `setCrashReportSupplier` was used for `method_1494`. liach's suggestion is to use this name `setCrashReportSupplier` (or better `setCrashReport`) for `method_43587`. Otherwise, it's confusing because `setCrashReportWithoutDetails` sounds like it removes something from the existing crash report, which is not the case.

@Shnupbups @liach Is my understanding correct?

Reply:
```suggestion
	METHOD method_43587 setCrashReportSupplier (Lnet/minecraft/class_128;)V
```